### DISCUSSION TITLE
Stateful TreeTable?

### DISCUSSION THREAD

**Discussion by MBelniak** [primefaces/community#1329] [PrimeReact] [answered]
DataTable can save its configuration in session/local storage https://primereact.org/datatable/#stateful
How difficult would it be to implement the same functionality in TreeTable? I assume most of the logic can be borrowed form the DataTable.

**Comment by melloware** [comment]
Yep i assume also you could try to submit a PR supporting that feature?  If not you would probably have to financially sponsor it via PrimeReact PRO support.

**Reply by MBelniak** [reply, marked as answer]
If there are no plans on doing it I can provide a PR at some point in time.
